### ISSUE TITLE
pageChange event giving payload as undefined.

### ISSUE THREAD

**Issue by devd251993** [SSENSE/vue-carousel#245]
I want to display current image number, that user is watching. I have included buttons, on clicking then image number changes. I am trying to listen pageChange event, which should give me current page number. The vue-dev tools event tab showing me correct payload, but when I try to access it in my code, the value of payload is undefined. 
My configuration is as follows:
 "vue": "^2.5.7",
 "vue-carousel": "^0.9.0",

<b>HTML Code:</b>
[code.txt](https://github.com/SSENSE/vue-carousel/files/2205515/code.txt)

<b>Vue.js Functions:</b>
updateNavigationNumber(number){
        console.log('Index',number)
   }

Output of Vue-devtools:
![image](https://user-images.githubusercontent.com/6411651/42877336-f071690a-8aa6-11e8-8632-48ef857e0236.png)

Console output:
![image](https://user-images.githubusercontent.com/6411651/42877372-0d91fab8-8aa7-11e8-9a57-8ca80a14cd8c.png)

I dont know where I am doing it wrong. Please guide me through.
P.S.: I am new to Vue.js

**Comment by quinnlangille** [comment]
Hey @devd251993, at first glance I'd try changing the case of the event to all lowercase `@pagechange` vs `@pageChange`. That could be causing the issue. 

If that doesn't work, there's an example of this event in our `vue-play` instance that is working, here's the source for it:

```
    template:
      `<div style="width: 100%; display: flex; justify-content: center; margin-top: 40px;">
        <carousel style="width: 500px;" @pagechange="onPageChange">
          <slide v-for="slide in slides">
            <img style="width: 100%;" :src="slide" />
          </slide>
        </carousel>
      </div>`,
    components: {
      Carousel,
      Slide
    },
    data() {
      return {
        slides: images
      }
    },
    methods: {
      onPageChange(currentPage) {
        this.$log(`page changed to ${currentPage}`)
      },
    }
  })
```

You can also find this example running on the dev server of the project at `localhost:5000`.

If after taking a look at the example you find you're still having the issue, feel free to post back here and I'll see if I can investigate :~)

Lemme know if the case change fix works!

**Comment by devd251993** [comment]
@quinnlangille  Thank you for reply. The case change does not work. There was error in my code only.
I changed `@pageChange="updateNavigationNumber(number)" `to `@pageChange="updateNavigationNumber"` and it worked. And also in play example `@pagechange` wont work. After changing it to `@pageChange`  I started getting logged values.  For `@pagechange`  I got following error 
![image](https://user-images.githubusercontent.com/6411651/42887327-59508854-8ac3-11e8-9063-b395be44dd22.png)
